examples/lightning_base.py

```diff
@@ -217,28 +199,26 @@ def add_model_specific_args(parser, root_dir):
 
 
 class LoggingCallback(pl.Callback):
+    @rank_zero_only
```

Comment:
this is not correct  (nor needed)... all hooks in a lightning module are called on the right devices.
on_validation_end needs to be called on every machine not just rank 0

Reply:
But we only want to write to disk from the master process, right?
If `LoggingCallback.on_validation_end` is called on every process, won't we write to disk n times?